tt_metal/hw/inc/bfloat16_utils.h

```diff
@@ -0,0 +1,39 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <stdint.h>
+
+// Optimized function to compare two bfloat16 values using integer arithmetic
+bool bfloat16_greater(uint16_t bf16_a, uint16_t bf16_b) {
```

Comment:
Is this a fn that will be used on the device or host?  If host, it doesn't belong here.  If device then...this isn't an api but a utility function.  We don't have our device incs factored well for this, but I suggest we put it under hw/inc/utils/bfloat16.h and get included with "utils/blfoat16.h"

Reply:
Device. Good suggestion, moved.